### DISCUSSION TITLE
Breadcrumbs

### DISCUSSION THREAD

**Discussion by madsjoh** [dxc-technology/halstack-react#1623] [Ideas] [answered]
Implement a new breadcrumbs component to improve navigation.

It would also be nice to have the ability to have a context menu for each breadcrumb just like it's done in the Jenkins UI.

![bc](https://github.com/dxc-technology/halstack-react/assets/9995017/a3e7194d-9230-478c-aa1c-75f3fe5a84f0)

FYI, as of react-router v6.4, there is an officially supported way of producing breadcrumbs: https://reactrouter.com/en/main/hooks/use-matches#breadcrumbs.

**Comment by GomezIvann** [comment, marked as answer]
Hi @madsjoh.

I completely agree. We already have the Breadcrumbs component in our roadmap and we will address it soon. We will keep you updated on the component implementation.

Thanks for the feedback.

**Reply by mjohannesse3** [reply]
@GomezIvann thanks for the info
